week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,9 +7,18 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
-
+function convertToUSD(price) {
+  var result = price * 1.4;
+  var conversionFee = result * 0.01;
+  result += conversionFee;
+  return result.toFixed(2);
+}
+function convertToBRL(price) {
+  var result = price * 4.93;
+  var conversionFee = result * 0.01;
+  result += conversionFee;
+  return result.toFixed(2);
+}
```

Comment:
Both give you the correct result, but have you heard about the [DRY principle?](https://www.google.pl/search?q=dry+principle&oq=DRY+prin&aqs=chrome.0.0j69i57j0l4.2694j0j4&sourceid=chrome&ie=UTF-8)

The `0.01` fee is the same for the US and Brazilian market, and the `convertToUSD` and `convertToBRL` function names don't suggest that they'll add a fee as well - just the conversion of currency. So feel free to add an extra function and call it from somewhere else :)

Reply:
@martynamaron, Noted will work on that, thanks